WebApp/autoreduce_webapp/reduction_viewer/views.py

```diff
@@ -536,11 +538,6 @@ def stats(_):
     return context_dictionary
 
 
-# Constants that define the type of user that submitted a run
-DEVELOPMENT_TEAM_SUBMISSION = -1  # For example: using manual_submission.py
-AUTOREDUCTION_SERVICE_SUBMISSION = 0  # A user submits a run
-
-
```

Comment:
Not sure why these were removed?

Reply:
After your comments I thought that it was out of the scope of this PR to add in constants that replace -1 and 0. Here's the master branch and new started_by_id_to_name() as it is currently.

Master:
```python
    name = None
    if started_by_id is not None:
        if started_by_id == -1:
            name = "Development team"
        elif started_by_id == 0:
            name = "Autoreduction service"
        elif started_by_id > 0:
            try:
                user_record = User.objects.get(id=started_by_id)
                name = f"{user_record.first_name} {user_record.last_name}"
            except ObjectDoesNotExist as exception:
                LOGGER.error(exception)
                name = None
        elif started_by_id < -1:
            name = None
    return name
```

New:
```python
    if started_by_id is None or started_by_id < -1:
        return None

    if started_by_id == -1:
        return "Development team"

    if started_by_id == 0:
        return "Autoreduction service"

    try:
        user_record = User.objects.get(id=started_by_id)
        return f"{user_record.first_name} {user_record.last_name}"
    except ObjectDoesNotExist as exception:
        LOGGER.error(exception)
        return None
```